cvereporter/report.py

```diff
@@ -0,0 +1,50 @@
+from cyclonedx.exception import MissingOptionalDependencyException
+from cyclonedx.factory.license import LicenseFactory
+from cyclonedx.model import OrganizationalEntity, XsUri
+from cyclonedx.model.bom import Bom
+from cyclonedx.model.component import Component, ComponentType
+from cyclonedx.model.impact_analysis import ImpactAnalysisAffectedStatus
+from cyclonedx.model.vulnerability import Vulnerability, VulnerabilitySource,VulnerabilityScoreSource, VulnerabilityRating, VulnerabilitySeverity, BomTarget, BomTargetVersionRange
+from cyclonedx.output import make_outputter, LATEST_SUPPORTED_SCHEMA_VERSION
+from cyclonedx.output.json import JsonV1Dot4
+from cyclonedx.schema import SchemaVersion, OutputFormat
+from cyclonedx.validation.json import JsonStrictValidator
+from cyclonedx.validation import make_schemabased_validator
+from datetime import datetime
+
+# based on sample code from https://cyclonedx-python-library.readthedocs.io/en/latest/examples.html
+lc_factory = LicenseFactory()
+bom = Bom()
```

Comment:
Missing some of the head info in the original proposal that would be good to have: schema, bomformat, spec version, etc

Reply:
i don't see these in the constructor,  they may be there under a different name